lib/watcher_adapter.js

```diff
@@ -27,12 +28,13 @@ module.exports = class WatcherAdapter extends EventEmitter {
     if (!Array.isArray(watchedPaths)) {
       throw new TypeError(`WatcherAdapter#watch's first argument must be an array of watchedPaths`);
     }
-    let watchers = watchedPaths.map(watchedPath => {
+    let watchers = watchedPaths.map(({ root: watchedRoot, path: watchedPath }) => {
```

Comment:
I think this should be called `watchedNode` rather than `root` as that to be signifies a path root

Reply:
Sounds good. I will update